app/services/events/post_process_service.rb

```diff
@@ -38,6 +38,10 @@ def call
     rescue ActiveRecord::RecordInvalid => e
       delivor_error_webhook(error: e.record.errors.messages)
 
+      result
```

Comment:
Lint/UselessAssignment: Useless assignment to variable - `e`. (https://rubystyle.guide#underscore-unused-vars)